src/main/java/Command.java

```diff
@@ -0,0 +1,80 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Command {
```

Comment:
What does this class specifically command? What name should this be?

Reply:
> What does this class specifically command? What name should this be?

Emm, it's mainly because in my early implementation, I only trim the user input into tokens without keeping original user input, but then I found some requirements actually need the original user input so I wrapped both into "Command". Thanks for reviewing.